src/App.tsx

```diff
@@ -192,6 +251,17 @@ export default function App() {
               {results && (
                 <ReactJson src={results} enableClipboard={true} theme="shapeshifter:inverted" collapsed={2} />
               )}
+              {results && (
+                <Button
+                  variant="contained"
+                  color="primary"
+                  onClick={() => {
+                    fileDownload(JSON.stringify(results), `results-${measureFileName}.json`);
```

Comment:
I think this will actually make it like `results-EXM130-7.3.000-bundle.json.json`. Maybe we can handle when there's a file extension and when there isn't? 

Like check if the measureFileName string includes the pattern `.json`. If it does, then we don't add. our own. if it doesn't, then we do?